docs/source/deploy.rst

```diff
@@ -227,13 +227,11 @@ unix socket:
     After=network.target
 
     [Service]
-    PIDFile=/run/gunicorn/pid
     User=someuser
     Group=someuser
     RuntimeDirectory=gunicorn
     WorkingDirectory=/home/someuser/applicationroot
-    ExecStart=/usr/bin/gunicorn --pid /run/gunicorn/pid   \
-              --bind unix:/run/gunicorn.sock applicationname.wsgi
+    ExecStart=/usr/bin/gunicorn applicationname.wsgi
```

Comment:
there's no additional settings or config file.

the removed `--pid` option is because it's not needed. the same reason why the unit file PIDFile is removed.

the `--bind` argument is not needed for this example, since the gunicorn process will inherit the socket file descriptor from systemd socket activation.

ps. of course I have tested this :)

Reply:
@gdamjan not against that change but maybe it worth in this case to let the users know how they can fetch the main  pid where to use the signals manually? or at least we may want a ticket for it :)

Reply:
it depends how deep we want to make the docs. here are at least 2 possibilities that I can think out of my head:
`systemctl show --value -p MainPID gunicorn.service`, but even better is `systemctl kill -s HUP gunicorn.service`

Reply:
@gdamjan having a line or two for it would be good, so the user will not have to look for :-)